site/content/blog/2022-03-01-whats-new-in-svelte-march-2022.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "What's new in Svelte: March 2022"
+description: "Shadow endpoints and update detection come to SvelteKit"
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+Some long-requested features have been added to SvelteKit this month... including shadow endpoints! This change in how `load` functions makes it easier to fetch data required for basic pages, redirect from POST responses and handle 404s and other errors.
+
+More on that and other new features and fixes below!
+
+## New across Svelte and Language Tools
+- Any highlights from the Svelte Changelog? Feels like it was mostly bugfixes
+...
+- svelte.dev is [now available in Russian](https://ru.svelte.dev/)! If you'd like to contribute to the community transaltions, check out [sveltejs-translations/content-server](https://github.com/sveltejs-translations/content-server)
+- Accessing properties in markups has been improved in the Svelte language tools ([105.12.0](https://github.com/sveltejs/language-tools/releases/tag/extensions-105.12.0)) - working around some known issues with autocomplete ([#538](https://github.com/sveltejs/language-tools/issues/538), [#786](https://github.com/sveltejs/language-tools/issues/786) and [#1302](https://github.com/sveltejs/language-tools/issues/1302))
```

Comment:
I think we don't have to reference 786 here. That's an implementation PR related to this new improvement instead of a known issue. People interested in the implementation detail could still find it through the release note. 

Another somewhat related to language tools that might be worth noting is that [svelte-atom](https://github.com/sveltejs/svelte-atom) is back 🚀. Kudos to the effort by rixo.